tests/Feature/ResourceFactoryTest.php

```diff
@@ -490,20 +490,26 @@ public function testCreateShipment()
     {
         $resourceFactory = new ResourceFactory();
         $shipment = $resourceFactory->create('shipments', [
-            'barcode'                 => 'S3BARCODE',
-            'description'             => 'order #012ASD',
-            'price'                   => 99,
-            'currency'                => 'USD',
-            'insurance_amount'        => 50,
-            'weight'                  => 8000,
-            'physical_properties'     => [
+            'barcode'                      => 'S3BARCODE',
+            'description'                  => 'order #012ASD',
+            'price'                        => 99,
+            'currency'                     => 'USD',
+            'insurance_amount'             => 50,
+            'physical_properties'          => [
                 'weight' => 1000,
                 'length' => 1100,
                 'volume' => 1200,
                 'height' => 1300,
                 'width'  => 1400,
             ],
-            'recipient_address'       => [
+            'physical_properties_verified' => [
+                'weight' => 100,
+                'length' => 110,
+                'volume' => 120,
```

Comment:
Incorrect example `volume` might confuse people reading the tests to check how the class should work.